src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```

Comment:
don't reset for all it is a very difficult process. Write tags which you have reset.